docs/dockstore-logo-usage-guidelines.rst

```diff
@@ -0,0 +1,129 @@
+
+.. |Vertical Black Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-black-blue.png
+  :width: 128
+
+.. |Horizontal White Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-white-blue.png
+  :width: 256
+
+.. |Horizontal Black Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-black-blue.png
+  :width: 256
+
+.. |Icon Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-blue.png
+  :width: 64
+
+.. |Icon Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-black.png
+  :width: 64
+
+.. |Icon White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-white.png
+  :width: 64
+
+===============================
+Dockstore Logo Usage Guidelines
+===============================
+
+About
+=====
+It is important to establish a sense of branding throughout our publicly facing, digital media. Currently, we have several themes / layouts, navigation systems, font sets, color palettes. We would like to unify our designs under a single set of standards.
+
+Color
+=====
+The official Dockstore color is:
+
+  .. list-table:: Dockstore Blue
+
+    * - |Dockstore Dot|
+      - Hex: #7986CB
+      - R: 121, G: 134, B: 203
+
+When using the Dockstore logo or icon, the acceptable colors are Dockstore Blue, black, or white as shown in the “Logo” section of this document.
+
+If the surrounding color is visually unappealing with Dockstore Blue, please use the black or white variants as appropriate.
+
+Logos
+=====
+
+  .. tabularcolumns:: |c|c|c|
+
+  .. list-table::
+    :header-rows: 1
+
+    * - Horizontal
+      - Vertical
+      - Icon
+    * - |Horizontal blue|
+      - |Vertical blue|
+      - |Icon blue|
+    * - Dockstore-logo-horizontal-blue
+      - Dockstore-logo-vertical-blue
+      - Dockstore-icon-blue
+    * - |Horizontal black|
+      - |Vertical black|
+      - |Icon black|
+    * - Dockstore-logo-horizontal-black
+      - Dockstore-logo-vertical-black
+      - Dockstore-icon-black
+    * - |Horizontal white|
+      - |Vertical white|
+      - |Icon white|
+    * - Dockstore-logo-horizontal-white
+      - Dockstore-logo-vertical-white
+      - Dockstore-icon-white
+    * - |Horizontal black blue|
+      - |Vertical black blue|
+      -
+    * - Dockstore-logo-horizontal-black-blue
+      - Dockstore-logo-vertical-black-blue
+      -
+    * - |Horizontal white blue|
+      - |Vertical white blue|
+      -
+    * - Dockstore-logo-horizontal-white-blue
+      - Dockstore-logo-vertical-white-blue
+      -
+
+A zip file containing these logos in both PNG and SVG formats can be found :download:`here </assets/images/Dockstore_Logos_and_Icons.zip>`.
+
+Use
+===
+Do:
+
+* Use the logo or icon color that works best in the context of the document, web page, or application.
+* The Dockstore logo is preferred.  Use the Dockstore icon only when space is limited or when it is clear that the reference is to Dockstore.
+* Spacing:
```

Comment:
I know you mentioned you updated the formatted list from what's shown here, but in case you want to switch back to nested lists, you can try adding an extra newline after each bullet (including the parent)

<img width="910" alt="Screen Shot 2022-07-07 at 4 13 34 PM" src="https://user-images.githubusercontent.com/27784612/177886521-0d6c8c06-c156-453c-ac5f-44c48fbdcfc3.png">

Reply:
This was a mistake.  Accidentally deleted the blank line.